pkg/userdata/helper/kubelet.go

```diff
@@ -279,8 +297,33 @@ func kubeletConfiguration(clusterDomain string, clusterDNS []net.IP, featureGate
 }
 
 // KubeletFlags returns the kubelet flags.
-func KubeletFlags(version, cloudProvider, hostname string, dnsIPs []net.IP, external bool, pauseImage string, initialTaints []corev1.Taint, extraKubeletFlags []string) (string, error) {
-	tmpl, err := template.New("kubelet-flags").Funcs(TxtFuncMap()).Parse(kubeletFlagsTpl)
+func KubeletFlags(version, cloudProvider, hostname string, dnsIPs []net.IP, external bool, ipFamily util.IPFamily, pauseImage string, initialTaints []corev1.Taint, extraKubeletFlags []string) (string, error) {
+	// --node-ip and --cloud-provider kubelet flags conflict in the dualstack setup.
```

Comment:
Probably this documentation should be written on top of the func since you are describing the effect of parameter `ipFamily`

Reply:
Makes sense. Will add to the function doc.